InternProject/DAL/Models/DeliveryTrack/DCBookingModel.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace InternProject.DAL.Models.DeliveryTrack
+{
+    public class DCBookingModel
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [StringLength(20)]
+        public string DeliveryMethod { get; set; }
+
+        [StringLength(50)]
+        public string WareHouse { get; set; }
+
+        [StringLength(12)]
+        public string BookingRef { get; set; }
+
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:MM/dd/yyyy}")]
+        public DateTime BookingDate { get; set; }
+
+        [StringLength(12)]
+        public string BookingTime { get; set; }
+
+        [StringLength(30)]
+        public string Haulier { get; set; }
+
+        [StringLength(30)]
+        public string CreatedBy { get; set; }
+
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:MM/dd/yyyy}")]
```

Comment:
This is data annotation attribute for ASP.NET MVC only, no effect for EF

Reply:
remove data annotations that don't affect EF